### PR TITLE
framework: improve router error messages

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: framework/resource_router.go [hunk 1]
```diff
@@ -37,10 +37,10 @@ func (r *ResourceRouter) ResourceSet(obj interface{}) (*ResourceSet, error) {
 	}
 
 	if len(found) == 0 {
-		return nil, microerror.Maskf(executionFailedError, "could not find any resource router")
+		return nil, microerror.Maskf(executionFailedError, "could not find handling resource set")
```
Comment (tuommaki) [inline framework/resource_router.go]: Nitpick, but for me this somehow sounds weird. What would think of following:
`no match for resource set`

File: framework/resource_router.go [hunk 2]
```diff
@@ -37,10 +37,10 @@ func (r *ResourceRouter) ResourceSet(obj interface{}) (*ResourceSet, error) {
...
 	}
 	if len(found) > 1 {
-		return nil, microerror.Maskf(executionFailedError, "multiple resource routers not allowed")
+		return nil, microerror.Maskf(executionFailedError, "multiple handling resource sets found")
```
Comment (tuommaki) [inline framework/resource_router.go]: Along the lines of above, my suggestion for this:
`multiple matches for resource set; only single allowed`

File: framework/resource_router.go (hunk 1 above)
Comment (kopiczko) [reply framework/resource_router.go]: This means that resource set that is capable of handling custom object that just arrived wasn't found.

I guess is sounds more reasonable when you are into the details, doesn't it?

I'd be glad to make it something reasonable for a newcomer. `handling resource set not found` makes more sense?

Comment (tuommaki) [reply framework/resource_router.go]: Yeah, I actually did know what this is used for. Your new suggestion sounds very good. I like it :+1:

File: framework/resource_router.go (hunk 2 above)
Comment (kopiczko) [reply framework/resource_router.go]: Changing to `multiple handling resource sets found; only single allowed` to align with `handling resource set not found`.

**Pull request merged**